### PR TITLE
Refactor environment check

### PR DESCRIPTION
**Pull request by danidoni**
In some of our development rake tasks we have several checks to ensure we only run those tasks against a development environment.

This PR DRY's that.

### CODE REVIEW

File: src/api/lib/tasks/dev.rake [hunk 1]
```diff
@@ -24,6 +24,15 @@ namespace :dev do
     RakeSupport.copy_example_file('../../dist/ec2utils.conf')
   end
 
+  task check_propper_environment: :environment do
```
Comment (dmarcoux) [inline src/api/lib/tasks/dev.rake:27]:
```suggestion
  task check_proper_environment: :environment do
```

File: src/api/lib/tasks/dev.rake [hunk 2]
```diff
@@ -24,6 +24,15 @@ namespace :dev do
     RakeSupport.copy_example_file('../../dist/ec2utils.conf')
   end
 
+  task check_proper_environment: :environment do
```
[the remark this answers is not in the document]
Comment (danidoni) [reply src/api/lib/tasks/dev.rake:27]: I had that at first, but the wording was weird to me: "Check development environment" looks like I'm checking if the development environment is ok, but what I want to check is that "The proper environment is selected". Maybe `check_proper_environment_set`?

[the remark this answers is not in the document]
Comment (eduardoj) [reply src/api/lib/tasks/dev.rake:27]: `proper` hides this has to do with the development environment.

What about simply `development_environment`?

[the remark this answers is not in the document]
Comment (danidoni) [reply src/api/lib/tasks/dev.rake:27]: okay

**Review by eduardoj — APPROVED** [review]

**Review by saraycp — COMMENTED** [review]
There are more tasks under `lib/tasks/dev/` that can be simplified in the same way.
But we can do it in another PR.

**Review by saraycp — APPROVED** [review]

**Comment by danidoni** [comment]
> There are more tasks under `lib/tasks/dev/` that can be simplified in the same way. But we can do it in another PR.

True, I forgot to add them all. Stay tuned for upcoming changes

**Comment by danidoni** [comment]
[Enhance BsRequest development data: Part II (L)](https://trello.com/c/ZCzRdEyg/2095-enhance-bsrequest-development-data-part-ii-l)

**Review by saraycp — APPROVED** [review]

**Pull request merged**